src/controllers/firewalls/firewall.controller.ts

```diff
@@ -236,4 +237,38 @@ export class FirewallController extends Controller {
             throw error;
         }
     }
+
+    @Validate()
+    async configCommunication(request: Request): Promise<ResponseBuilder> {
```

Comment:
We can provide this value when the UI is getting the data about the `fwcloud` (as this config value is `fwcloud`-related config parameter) and save this extra request.